index.js

```diff
@@ -75,11 +73,10 @@ var PersonaClient = function (config) {
 /**
  * Validate bearer token against cache or persona
  * @param token: Token to validate
- * @param scope: Requested scope
- * @param overrideScope: Backup scope to use instead of scope
```

Comment:
I don't know why this is here. If anyone can give me a good reason to keep it I'll consider it. But I think validating against one or more tokens as an array is cleaner.